cdk/package.json

```diff
@@ -48,19 +48,23 @@
     "@aws-sdk/client-cloudwatch": "3.525.0",
     "@aws-sdk/client-eventbridge": "3.525.0",
     "@aws-sdk/client-organizations": "3.525.0",
+    "@aws-sdk/client-securityhub": "^3.536.0",
```

Comment:
Please don't use different versions of aws-sdk. We had typing issues in the past with this.